share/man/man9/cdefs.9

```diff
@@ -0,0 +1,283 @@
+.\"-
+.\" Copyright (c) 2024 M. Warner Losh <imp@FreeBSD.org>
+.\"
+.\" SPDX-License-Identifier: BSD-2-Clause
+.\"
+.Dd July 3, 2024
+.Dt CDEFS 9
+.Os
+.Sh NAME
+.Nm cdefs
+.Nd "compiler portability macro definitions"
```

Comment:
```suggestion
.Nd compiler portability macro definitions
```
Please forgive me for wasting your time with such an irrelevant nit, but document descriptions do not need to be quoted literals. I have absolutely zero data to support my strong opinion that this unnece

Reply:
You're not wasting my time. Thanks

Reply:
updated